code/mod_table/mod_table.cpp

```diff
@@ -1372,6 +1373,10 @@ void parse_mod_table(const char *filename)
 				stuff_boolean(&Contrails_use_absolute_speed);
 			}
 
+			if (optional_string("$Scripted velocity:")) {
```

Comment:
This should be a little more verbose - `$Scripted velocity:` by itself is unclear.  Maybe `$Scripted velocity overrides physical velocity:`?

Reply:
Even when I have the words to spare, it's often still too complicated to try and explain exactly how it works, so I'm loathe to introduce more words that I feel like will only confuse further.